packages/cli-platform-ios/src/tools/prompts.ts

```diff
@@ -0,0 +1,34 @@
+import prompts from 'prompts';
```

Comment:
Not super sure about file name here...
There's also `promptForDeviceSelection` in there somewhere so we should either rename this file or move device selection prompt here. WDYT?

Reply:
good point, decided to move `promptForDeviceSelection` to this file to stay consistent